<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AL</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(61, 67, 87);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 245, 248);</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="dataButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(226, 230, 235,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/data.png</normaloff>:/icon/data.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="storageButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(226, 230, 235,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/storage.png</normaloff>:/icon/storage.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="leftButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(226, 230, 235,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/left.png</normaloff>:/icon/left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="rightButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(226, 230, 235,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/right.png</normaloff>:/icon/right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>D</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="informationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>199</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(61, 67, 87);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="matchButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(82,38,200);
color: rgb(226, 230, 235);
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: rgb(82,38,200);
padding: 4px;</string>
    </property>
    <property name="text">
     <string>Match</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(226, 230, 235,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/accept.png</normaloff>:/icon/accept.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rejectButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>670</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(226, 230, 235,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icon/reject.png</normaloff>:/icon/reject.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="shortcut">
     <string>C</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="dataList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>150</width>
      <height>190</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(61, 67, 87);</string>
    </property>
   </widget>
   <widget class="QLabel" name="informationLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(61, 67, 87);</string>
    </property>
    <property name="text">
     <string># of Data in Dir</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="confThLabel_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(226, 230, 235);</string>
    </property>
    <property name="text">
     <string>Confidence Th </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="confThLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(94, 226, 177);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="AL">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>740</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/icon/icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imgName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(226, 230, 235);</string>
    </property>
    <property name="text">
     <string>  Img Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="iouThLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(61, 67, 87);</string>
    </property>
    <property name="text">
     <string>IoU th</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="iouSpin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="confThLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(61, 67, 87);</string>
    </property>
    <property name="text">
     <string>Conf th</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="confSpin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
   </widget>
   <widget class="QPushButton" name="saveOnceButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(25,204,224);
color: rgb(226, 230, 235);
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: rgb(25,204,224);
padding: 4px;</string>
    </property>
    <property name="text">
     <string>Save at Once</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>691</width>
      <height>851</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Inference</string>
     </attribute>
     <widget class="QLabel" name="humanAnno">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>8</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(226, 230, 235);</string>
      </property>
      <property name="text">
       <string>  Ground Truth</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="gtBG">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>27</y>
        <width>676</width>
        <height>386</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(61, 67, 87);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>30</y>
        <width>671</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="gtLayout"/>
     </widget>
     <widget class="QLabel" name="mIoULabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>800</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(94, 226, 177);</string>
      </property>
      <property name="text">
       <string>mIoU</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>417</y>
        <width>671</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="netLayout"/>
     </widget>
     <widget class="QLabel" name="autoAnno">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>800</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(226, 230, 235);</string>
      </property>
      <property name="text">
       <string>  Prediction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="net1BG">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>414</y>
        <width>676</width>
        <height>386</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(61, 67, 87);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mConfLabel">
      <property name="geometry">
       <rect>
        <x>485</x>
        <y>800</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(94, 226, 177);</string>
      </property>
      <property name="text">
       <string>mConf</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>net1BG</zorder>
     <zorder>humanAnno</zorder>
     <zorder>gtBG</zorder>
     <zorder>verticalLayoutWidget_3</zorder>
     <zorder>mIoULabel</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>autoAnno</zorder>
     <zorder>mConfLabel</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Autolabeling</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>7</y>
        <width>671</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="atLayout"/>
     </widget>
     <widget class="QLabel" name="atLabel">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>390</y>
        <width>140</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(226, 230, 235);</string>
      </property>
      <property name="text">
       <string>  Prediction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="atBG">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>4</y>
        <width>676</width>
        <height>386</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(61, 67, 87);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mConfLabel2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>390</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(61, 67, 87);
color: rgb(94, 226, 177);</string>
      </property>
      <property name="text">
       <string>mConf</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>atBG</zorder>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>atLabel</zorder>
     <zorder>mConfLabel2</zorder>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioInf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Inf</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioAuto">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(241, 245, 248);</string>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Acquisition_Helper"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout_Acquisition_Helper">
   <property name="text">
    <string>About AL</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
